queries.sql

```diff
@@ -66,4 +66,82 @@ WHERE owner_id = 5 AND escape_attempts = 0;
 SELECT full_name, COUNT(name) FROM owners o
 LEFT JOIN animals a
   ON a.owner_id = o.id
-GROUP BY full_name;
\ No newline at end of file
+GROUP BY full_name;
+
+-- QUESTIONS RELATED TO VISITS AND VETS - DAY 4
+
+SELECT v.name as vet, a.name as animal, vis.date_of_visit FROM visits vis
+INNER JOIN animals a
+  ON a.id = vis.animals_id
+INNER JOIN vets v
+  ON v.id = vis.vets_id
+WHERE vis.date_of_visit = (select max(date_of_visit) FROM visits WHERE vets_id=1) 
+ORDER BY vis.date_of_visit desc;
+
+SELECT v.name, a.name FROM visits vis
+INNER JOIN animals a
+  ON a.id = vis.animals_id
+INNER JOIN vets v
+  ON v.id = vis.vets_id
+WHERE v.name = 'Stephanie Mendez'
+GROUP BY (v.name, a.name);
+
+SELECT v.name, s.name FROM specializations sp
+INNER JOIN species s
+  ON s.id = sp.species_id
+RIGHT JOIN vets v
+  ON v.id = sp.vets_id;
+
+SELECT v.name, a.name, vis.date_of_visit FROM visits vis
+INNER JOIN animals a
+  ON a.id = vis.animals_id
+INNER JOIN vets v
+  ON v.id = vis.vets_id
+WHERE v.name = 'Stephanie Mendez' and date_of_visit BETWEEN 'April 1, 2020' and 'August 30, 2020';
+
+SELECT a.name, count(a.name) FROM visits vis
+INNER JOIN animals a
+  ON a.id = vis.animals_id
+INNER JOIN vets v
+  ON v.id = vis.vets_id
+GROUP BY a.name
+order by count desc;
```

Comment:
- [ ] The query question is what animal has the most visits to vets? Please ensure you only list the animal with the most visits